Calculator/Calculator/Model/Formula.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Formula.swift
+//  Calculator
+//
+//  Created by Danny on 2/11/24.
+//
+
+import Foundation
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double>
+    var operators: CalculatorItemQueue<Operator>
+    let defaultValue: Double = 0
```

Comment:
저는 개인적으로 상수는 따로 뺴서 사용하는 편입니다 ㅎㅎ
```suggestion
    enum Constant {
          static let zero = 0
    }
```

Reply:
상수를 별도 열거형 타입으로 정리 및 정의 사용하는 방법이 있는 줄은 몰랐습니다. 말씀해주신 내용은 잘 숙지하도록 하겠습니다. 😁